app/models/license.rb

```diff
@@ -1,13 +1,16 @@
-# typed: strict
+# typed: true
 # frozen_string_literal: true
 
-# Represents the list of valid work types
+# Class contains valid license types, their identifiers from https://spdx.org/licenses/
+#  and other structures used in the work form and its validation
 class License
   extend T::Sig
 
+  # identifier from https://spdx.org/licenses/
   sig { returns(String) }
   attr_reader :id
 
+  # displayed to user in work form
   sig { returns(String) }
   attr_reader :label
```

Comment:
1. does it make sense to keep `:label` as an attribute as we can simply derive it from the hash?
2. I assumed the work would keep the "id" (license code), and I did the validation a bit differently from before, without building all the diff License objects.  Is there any problem with this?

Basically - I wasn't sure what was supposed to happen next and if this approach will fooch that somehow.

Reply:
I think you should remove both methods as they are no longer in use.

Reply:
do we even want the constructor?

Reply:
We don't need it since you did away with the calls to it.